python/pyspark/mllib/stat/__init__.pyi

```diff
@@ -1,29 +0,0 @@
-#
-# Licensed to the Apache Software Foundation (ASF) under one
-# or more contributor license agreements.  See the NOTICE file
-# distributed with this work for additional information
-# regarding copyright ownership.  The ASF licenses this file
-# to you under the Apache License, Version 2.0 (the
-# "License"); you may not use this file except in compliance
-# with the License.  You may obtain a copy of the License at
-#
-#   http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing,
-# software distributed under the License is distributed on an
-# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
-# KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
-
-from pyspark.mllib.stat.KernelDensity import (  # noqa: F401
-    KernelDensity as KernelDensity,
-)
-from pyspark.mllib.stat._statistics import (  # noqa: F401
-    MultivariateStatisticalSummary as MultivariateStatisticalSummary,
-    Statistics as Statistics,
-)
-from pyspark.mllib.stat.distribution import (  # noqa: F401
-    MultivariateGaussian as MultivariateGaussian,
-)
-from pyspark.mllib.stat.test import ChiSqTestResult as ChiSqTestResult  # noqa: F401
```

Comment:
Is this safe to remove, @zero323?

Reply:
Seem like it @HyukjinKwon. There is nothing here, that we cannot pick out of `py` file.